packages/app/src/navigation/RootNavigation.tsx

```diff
@@ -16,6 +21,46 @@ const Stacks = () => {
       <Root.Screen name='PlanEdit' component={PlanInputScreen} />
       <Root.Screen name='Login' component={LoginScreen} />
       <Root.Screen name='Webview' component={WebviewScreen} />
+      <Root.Screen
+        name='AccountManage'
+        component={AccountManageScreen}
+        options={({ navigation }) => ({
+          title: '카카오 연동 계정 관리',
+          headerTitleAlign: 'left',
+          headerLeft: () => {
+            return (
+              <SvgXml
+                xml={icon_back}
+                style={{ marginLeft: 16 }}
+                fill={colors.black}
+                onPressOut={() => {
+                  navigation.goBack();
+                }}
+              />
+            );
+          },
```

Comment:
```suggestion
          headerLeft: () => <SvgXml
                xml={icon_back}
                style={{ marginLeft: 16 }}
                fill={colors.black}
                onPressOut={() => navigation.goBack()}
              />
```

depth가 너무 깊어지는 것 같아서, 의미없는 부분은 제거하는 게 어떨까용?